Monaco editor - New Control

joaojmendes opened the pull request:
| Q               | A
| --------------- | ---
 
| New feature?    | [X]

 

#### What's in this Pull Request?

New Control Monaco Editor

AJIXuMuK commented:
@joaojmendes - could you please check the `package.json`?
You're importing `monaco-editor` but I don't see it referenced.
And as a result:
<img width="1365" alt="Screen Shot 2022-03-05 at 5 24 16 PM" src="https://user-images.githubusercontent.com/17036219/156901723-c40a766d-7382-4e7c-97c2-2bb9d3a8ea5e.png">

joaojmendes commented:
Hi @AJIXuMuK ,  I updated my branch with the latest "master", and added the missing package. I tested and for me everything is working.
<img width="1975" alt="Screenshot 2022-03-06 at 15 17 01" src="https://user-images.githubusercontent.com/12551125/156930098-d3af17ea-1bc1-437a-9d66-930d1560c9a4.png">
<img width="1605" alt="Screenshot 2022-03-06 at 15 25 01" src="https://user-images.githubusercontent.com/12551125/156930108-57354981-b5e6-4f4f-aff5-db6ffe96b9b7.png">

Please tell me if everything is ok.
Thank you very much
